src/components/admin/form-components/validated-fields.jsx

```diff
@@ -17,15 +17,18 @@ export const cannotNull = value => {
   return null;
 };
 
-export const hasVarCharMax = value => {
-  if (value.length > 150) {
-    return 'The database restricts this field to a maximum value of 150 characters.';
+export const shorterThanDbVarCar = value => {
```

Comment:
`VarChar`* I see where you get your pronunciation from now ;)

Reply:
Haha typos model life! My pronunciation comes from the fact that the word it represents is pronounced "K-aracter" not literally "CH-aracter".

Reply:
That makes sense actually haha

Reply:
https://english.stackexchange.com/questions/60154/how-to-pronounce-the-programmers-abbreviation-char

everyone is right :P

Reply:
I'm the naive one though haha

Reply:
I'm in Noam Chomsky's camp when it comes to pronunciation and grammar! If it's widely understood, it's not incorrect 😁. So usually in pronunciation of made-up or heavily modified words, nobody is wrong!